format/mp4/boxes.go

```diff
@@ -1367,8 +1367,45 @@ func decodeBox(ctx *decodeContext, d *decode.D, typ string) {
 		d.FieldU32("h_spacing")
 		d.FieldU32("v_spacing")
 	case "uuid":
-		d.FieldRawLen("uuid", 16*8, scalar.RawUUID, uuidNames)
-		d.FieldRawLen("data", d.BitsLeft())
+		uuidBR := d.FieldRawLen("uuid", 16*8, scalar.RawUUID, uuidNames)
+		// TODO: make nicer
+		uuid := d.ReadAllBits(uuidBR)
+
+		switch {
+		case bytes.Equal(uuid, uuidTfxdBytes[:]):
+			version := d.FieldU8("version")
+			d.FieldU24("flags")
+			d.FieldStruct("fragment", func(d *decode.D) {
+				if version == 1 {
+					d.FieldU64("absolute_time")
```

Comment:
Does these timestamp have an epoch? if so they could use something like `scalar.UintActualUnixTimeDescription(time.Second, time.RFC3339)` but don't see anything in the spec

Reply:
It doesn't have any epoch. It represents time in timescale increments for the track.